services/convert/git_commit.go

```diff
@@ -198,8 +207,17 @@ func ToCommit(ctx context.Context, repo *repo_model.Repository, gitRepo *git.Rep
 		affectedFileList := make([]*api.CommitAffectedFiles, 0, len(fileStatus.Added)+len(fileStatus.Removed)+len(fileStatus.Modified))
 		for _, files := range [][]string{fileStatus.Added, fileStatus.Removed, fileStatus.Modified} {
 			for _, filename := range files {
+				var filestatus = "unknown"
+				if contains(fileStatus.Added, filename) {
```

Comment:
You should use `util.SliceContainsString` directly.

Reply:
Thanks. I updated the code accordingly